src/plugins/git/types.js

```diff
@@ -55,6 +55,27 @@ export type NodeType =
 
 // Edges
 
+// CommitNode -> CommitNode
+export const HAS_PARENT_EDGE_TYPE: "HAS_PARENT" = "HAS_PARENT";
+export type HasParentEdgePayload = {|
+  +parentIndex: number, // one-based
+|};
+export function hasParentEdgeId(
+  childCommitHash: Hash,
+  oneBasedParentIndex: number
+) {
+  if (
+    oneBasedParentIndex !== (oneBasedParentIndex | 0) ||
```

Comment:
Math.floor

Reply:
Sure; done, and added the necessary `isFinite` check.